src/newsletter-snackbar/newsletter-snackbar.js

```diff
@@ -15,6 +14,10 @@ export default function NewsletterSnackbar() {
 
     setOpen(false);
   };
+  useEffect(() => {
```

Comment:
[React hooks must be called before anything else within the component](https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level) so this needs to be before `handleClose`.